orm_lib/inc/drogon/orm/SqlBinder.h

```diff
@@ -288,7 +289,8 @@ class SqlBinder
               size_t sqlLength,
               DbClient &client,
               ClientType type)
-        : sqlViewPtr_(sql),
+        : sqlPtr_(std::make_shared<std::string>(sql, sqlLength)),
```

Comment:
I think we should delay the memory copy to the moment when changing the SQL statement.

Reply:
Yeah, in spirit of c++, you should only pay for what you use